ompi/mca/op/rocm/op_rocm.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2019-2023 The University of Tennessee and The University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#ifndef MCA_OP_CUDA_EXPORT_H
```

Comment:
should this be MCA_OP_ROCM_EXPORT_H ?

Reply:
That was an artefact of `hipify`. Should be fixed in latest push.